spec/classes/puppet_spec.rb

```diff
@@ -139,6 +139,25 @@
         end
       end#no params
 
+      context 'when allinone is true' do
+        let(:params) {{'allinone' => true}}
+        it 'should manage the facts directories' do
```

Comment:
This is misleading.
We already have a test for this.
You should move this as a context inside the other test.
You will need to add a context around the current one as well.